app/App.tsx

```diff
@@ -78,8 +79,12 @@ const App: FC = () => {
         // solves an issue of this component's local state fighting with the redux store.
         const { selectedAccount } = store.getState();
         if (selectedAccount !== initialReduxStoreState.selectedAccount) {
-          await fetchBalance(selectedAccount);
-          await fetchLogs(selectedAccount);
+          // check wallet status to see if we need to update account
+          const isSynced = await isAccountSynced({ accountId: selectedAccount.account.accountId });
```

Comment:
before this would always fetch the logs and balance every 10 seconds. Now we check to see if the account is sycned and only fetch logs and balance if we are not synced